File: Back/src/Tokenlab.API/Models/Evento.cs
namespace Tokenlab.API.Models
{
    public class Evento
    {
        public int EventoId { get; set; }
        public string Tema { get; set; }
        public string Descricao { get; set; }
        public string Local { get; set; }
        public string DataEvento { get; set; }
        public string HoraInicio { get; set; }
        public string HoraTermino { get; set; }
        public int QtdPessoas { get; set; }
        public string Lote { get; set; }
        public string ImagemURL { get; set; }
    }
}

File: Front/Tokenlab-App/src/app/models/Evento.ts
import { RedeSocial } from './RedeSocial';
import { Lote } from "./Lote";
import { Palestrante } from './Palestrante';

export interface Evento {
  id: number;
  tema: string;
  descricao: string;
  local: string;
  dataEvento?: Date;
  horaInicio: string;
  horaTermino: string;
  qtdPessoas: number;
  imagemURL: string;
  telefone: string;
  email: string;
  lotes: Lote[];
  redesSociais: RedeSocial[];
  palestrantesEventos: Palestrante[];
}
